core/src/types/blocking_read/std_bytes_iterator.rs

```diff
@@ -28,7 +29,7 @@ use crate::raw::*;
 ///
 /// StdIterator also implements [`Send`] and [`Sync`].
 pub struct StdBytesIterator {
-    inner: oio::BlockingReader,
```

Comment:
Please change `BlockingReader` directly: Make itself `Arc<dyn oio::BlockingRead>`.

Reply:
I submit a new, like this?

Reply:
Thanks, this is what I want.